resources/views/inputs/checkbox.blade.php

```diff
@@ -26,7 +27,7 @@ class="form-checkbox input-checkbox"
             />
         </div>
 
-        <div class="pl-7 text-sm leading-5">
+        <div class="{{ $labelWrapperClass }}">
```

Comment:
if you need to be able to change the alignment of the box, it's better to have that as a predefined property (e.g. `alignment="left"` and switch classes based on that. Otherwise the alignment will become inconsistent when different people set their own classes for similar changes.

regarding the other styling needs, are those possible with the existing `labelClasses` property? as that touches the label itself